src/pages/studyView/table/ClinicalTable.tsx

```diff
@@ -86,9 +86,9 @@ export default class ClinicalTable extends React.Component<IClinicalTableProps,
             <span>{getFrequencyStr((data.count / this.totalCount) * 100)}</span>,
         tooltip: (
             <span>Percentage of {this.props.patientAttribute ? 'patients' : 'samples'}</span>),
-        filter: (d: ClinicalDataCountWithColor, f: string, filterStringUpper: string) => {
+        filter: (d: ClinicalDataCountWithColor, f: string) => {
             let freq = getFrequencyStr((d.count / this.totalCount) * 100);
-            return (freq.indexOf(filterStringUpper) > -1)
+            return (freq.indexOf(f) > -1)
```

Comment:
we now have String.includes (and Array.includes) which is a little nicer than indexOf.  you don't need to fix all these, but going forward ...